elements/pfe-band/src/pfe-band--lightdom.scss

```diff
@@ -0,0 +1,36 @@
+@import "../../pfe-sass/pfe-sass";
+
+// For each section in the band layout
+@each $section in (aside, header, body, footer) {
```

Comment:
If we're just targeting edge here, should we wrap this in the `@include browser-query(edge)` ?

Reply:
Ah yes! I forgot nav merged so browser-query mixin is up for grabs! Added it here and to ie11 code below